CMakeLists.txt

```diff
@@ -57,6 +57,10 @@ find_package(Pango)
 find_package(GdkPixbuf)
 find_package(PAM)
 
+find_package(LibInput REQUIRED)
+add_definitions(${LIBINPUT_DEFINTIONS})
+include_directories(${LIBINPUT_INCLUDE_DIRS})
```

Comment:
The `add_definitions` lines has a typo and can be removed. Removing the other two lines causes me this when starting `./bin/sway -d &> log`:

```
./bin/sway: error while loading shared libraries: libinput.so.10: cannot open shared object file: No such file or directory
```